### PR TITLE
[CP] 3.16.1 release update and cherry picks

### PR DESCRIPTION
**Pull request by ianballou**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,233 +1,42 @@
-# 3.16.0 Tasty Taiyaki (2020-08-10)
+# 3.16.1 Tasty Taiyaki (2020-09-28)
```
Comment (jlsherrill) [inline CHANGELOG.md:1]: do we usually replace the entire changelog?

Comment (ianballou) [reply CHANGELOG.md:1]: Ah no, I'm guessing we shouldn't.  Will fix.

Comment (ianballou) [reply CHANGELOG.md:235]: Fixed!

**Review by chris1984 — APPROVED** [review]
ACK since @jlsherrill comments were addressed. Don't see anything else that looks out of place

**Pull request merged**